eclipselink/src/test/groovy/jpql_sort/QuerySortTest.groovy

```diff
@@ -148,7 +148,10 @@ class QuerySortTest extends DataSpec {
 
         then:
 
-        thrown(UnsupportedOperationException)
+        queryBuilder.getResultQueryString() == 'select u from sec$User u order by u.login asc, u.name desc, u.id asc'
```

Comment:
getResultQueryString() is called twice in this test. No need to call it in "when" section

Reply:
fixed tests